hazelcast/src/main/java/com/hazelcast/osgi/impl/Activator.java

```diff
@@ -45,17 +45,28 @@ public void start(BundleContext context)
         // Try to start javax.scripting - JSR 223
         activateJavaxScripting(context);
 
-        if (Boolean.getBoolean(HAZELCAST_OSGI_START)) {
-            hazelcastInstance = Hazelcast.newHazelcastInstance();
+        // TODO Is it possible that OSGI service is not null (deactivated) on start?
```

Comment:
Even though this case seems not possible by OSGI container, instead of `TODO`, I will throw `IllegalStateException`